src/board.py

```diff
@@ -18,3 +18,5 @@ def stringify_spaces(self):
             #Board.border 
         ]
     
+    def full(self):
+        return not " " in self.spaces
```

Comment:
What if you want to change the way an empty space is represented?

Reply:
Good point, I'll store it in a variable for the class to make it easier to change on a whim and make another pr. If the client wants the spaces represented in a completely different way then that's a discussion I need to have with them next time we chat

Reply:
This concern has been addressed in this PR:
https://github.com/Sillhouette/tic_tac/pull/8

Mind reviewing it real quick before I merge?

Reply:
I've made some adjustments to the infrastructure of this branch here: 

https://github.com/Sillhouette/tic_tac/pull/9

Please review for merge